tests/sentry/incidents/endpoints/test_serializers.py

```diff
@@ -780,7 +780,7 @@ def test_comparison_delta_below(self):
             context=self.context, instance=alert_rule, data=params, partial=True
         )
         assert serializer.is_valid(), serializer.errors
-        alert_rule = serializer.save()
+        alert_rule = cast(AlertRule, serializer.save())
```

Comment:
just want to make sure this is to appease mypy for validating the types later?

Reply:
yeah, weirdly it's thinking this is the wrong model because it inherits the snubaquery so I was getting errors like `"tuple[type[AlertRule], QuerySubscription]" has no attribute "alertruletrigger_set"`